Packs/ApiModules/Scripts/CoreXQLApiModule/CoreXQLApiModule.py

```diff
@@ -655,8 +655,9 @@ def start_xql_query_polling_command(client: CoreClient, args: dict) -> Union[Com
         # running the command again using polling with an interval of 'interval_in_secs' seconds.
         command_results = CommandResults()
         interval_in_secs = int(args.get('interval_in_seconds', 20))
+        timeout_in_secs = int(args.get('timeout_in_seconds', 600))
         scheduled_command = ScheduledCommand(command='xdr-xql-generic-query', next_run_in_seconds=interval_in_secs,
-                                             args=args, timeout_in_seconds=600)
+                                             args=args, timeout_in_seconds=timeout_in_secs)
```

Comment:
You are missing another place where this is needed. In `get_xql_query_results_polling_command`
```Python
    if outputs.get('status') == 'PENDING':
        demisto.debug(f"Returned status 'PENDING' for {args.get('query_id', '')}.")
        scheduled_command = ScheduledCommand(command='xdr-xql-get-query-results', next_run_in_seconds=interval_in_secs,
                                             args=args, timeout_in_seconds=600)
```

Reply:
@talzich not enough to only add to code, is this in the yml as well? Otherwise as far as i know the parameter will just be ignored when passing from scheduledcommand